homework-16/main.js

```diff
@@ -0,0 +1,118 @@
+window.addEventListener('DOMContentLoaded', () => {
+    const form = document.querySelector("form");
+    const search = document.querySelector(".header-search");
+    const moviesEl = document.querySelector(".movies");
+    const paginationEl = document.querySelector('.pagination');
+    let currentPage = 1;
+    let rows = 10;
+    form.addEventListener("submit", (e) => {
+        e.preventDefault();
+        getPage();
+
+    })
+
+    async function getPage(page) {
+        const resp = await fetch(`http://www.omdbapi.com/?apikey=f67b2711&s=${search.value}&page=${page}`);
+        const respData = await resp.json();
+        showMovies(respData);
+        displayPagination(respData.totalResults, rows);
+    }
+
+    function showMovies(data) {
+        moviesEl.innerHTML = "";
+        paginationEl.innerHTML = "";
+        if (!data.Search) {
+            moviesEl.innerHTML = "Sorry we cannot find movie";
+        }
+
+        data.Search.forEach(movie => {
+            const movieEl = document.createElement("div");
+            movieEl.classList.add("movie");
+            movieEl.innerHTML = `
+        <div class="movie">
+        <div class="movie-cover-inner">
+            <img src="${movie.Poster}"
```

Comment:
You should have a fallback image for posters to avoid errors

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/84010501/205029975-4341ce6b-6e6e-47cf-966e-3424d9dbf0ec.png">